### ISSUE TITLE
Force resolve cannot suppress errors for nullable param pass in field initializations.

### ISSUE THREAD

**Issue by nimakarimipour** [ucr-riple/NullAwayAnnotator#73]
**Describe the bug**
We currently support adding SuppressWarnings and NullUnmarked annotation to silence remaining errors through #69. After the outer loop completes, Annotator will run NullAway another time and surpasses all remaining errors with either `@NullUnmarked` or `@SuppressWarnings`. All remaining errors store their corresponding regions where that error is triggered. For all triggered regions where the enclosing method is not `"null"`, we mark them with `@NullUnmarked` annotation. Remaining errors are triggered in field declaration regions of classes. For field declarations, if they are assigned to `@Nullable` we suppress it by adding `@SupreessWarnings("NullAway")` and if they are nit initialized we add `@SupreessWarnings("NullAway.Init")`. This does not cover all cases, such as example below:

```java
class Foo {
     Bar f = new Bar(null);
}
``` 

In this case we will receive the error: `passing @Nullable where @Nonnull was expected` and the triggered regions are `class: Foo method: "null"`. This case is not covered in our current strategy. We should mark the called constructor with `@NullUnmarked`.

**To Reproduce**
Run with force resolve mode activated on the code below:
```java
class Foo {
     Bar f = new Bar(null);
}
```

**Expected behavior**
Mark `Bar(Object)` as `@NullUnmarked`.

**Stack trace**
Not-Applicable.

**OS (please complete the following information):**
 - OS: x.
 - Version: x. 
 - Annotator Version:  v1.3.3-SNAPSHOT.

**Additional context**
Not-Applicable.
